internal/stack/compose.go

```diff
@@ -146,6 +146,27 @@ func dockerComposeDown(options Options) error {
 	return nil
 }
 
+func dockerComposeStatus(options Options) ([]compose.ServiceStatus, error) {
+	p, err := compose.NewProject(DockerComposeProjectName, options.Profile.FetchPath(profile.SnapshotFile))
+	if err != nil {
+		return nil, errors.Wrap(err, "could not create docker compose project")
+	}
+
+	statusOptions := compose.CommandOptions{
+		Env: newEnvBuilder().
+			withEnv(stackVariantAsEnv(options.StackVersion)).
+			withEnvs(options.Profile.ComposeEnvVars()).
+			build(),
+		Services: options.Services,
+	}
+
+	statusServices, err := p.Status(statusOptions)
+	if err != nil {
+		return nil, errors.Wrap(err, "running command failed")
```

Comment:
There may be some errors that are ok for the status command, for example if a container is not found.

Reply:
As this method is based on docker-compose scenario to get the container IDs
```bash
docker-compose -f <path/compose.yml> ps -q elasticsearch kibana elastic-agent ...
```
in my tests I just could make it fail it when `elastic-package stack up` has not run or it has been run the `elastic-package stack down` command. So there is no compose scenario created, and therefore docker-compose do now know nothing.

Given that use case, as a first approach I've added an error message to help the developer:
```
 $ elastic-package stack status 
elastic-package has been installed.
2022/08/10 19:08:05 ERROR is Elastic stack created/running?
Error: failed getting stack status: running command failed: exit status 1
```

This failure is because the elasticsearch container/service is not found. Here the verbose output:

```
 $ elastic-package stack status -v 
2022/08/10 19:44:54 DEBUG Enable verbose logging
2022/08/10 19:44:54 DEBUG Distribution built without a version tag, can't determine release chronology. Please consider using official releases at https://github.com/elastic/elastic-package/releases
2022/08/10 19:44:54 DEBUG running command: /home/mariorodriguez/bin/docker-compose version --short
2022/08/10 19:44:54 DEBUG Services to check: [elasticsearch elastic-agent fleet-server kibana package-registry]
2022/08/10 19:44:54 DEBUG running command: /home/mariorodriguez/bin/docker-compose -f /home/mariorodriguez/.elastic-package/profiles/default/stack/snapshot.yml -p elastic-package-stack ps -q elasticsearch elastic-agent fleet-server kibana package-registry
no such service: elasticsearch
2022/08/10 19:44:54 ERROR is Elastic stack created/running?
Error: failed getting stack status: running command failed: exit status 1
```

Probably, it also could be reported that message and return an empty array of services, to avoid `stack status` command fail in that case.

Reply:
Just checked that in docker-compose v2 does not behave as in v1 in case more services are checked in `docker-compose ps`

Running this command:
```
elastic-package stack up -v -d --services=elasticsearch,kibana
```

Output for docker-compose v1 (1.29.2):
```
 $ elastic-package stack status -v

```

Output for docker-compose v2 (2.7.0):
```
 $ elastic-package stack status -v
2022/08/10 19:54:56 DEBUG Enable verbose logging
2022/08/10 19:54:56 DEBUG Distribution built without a version tag, can't determine release chronology. Please consider using official releases at https://github.com/elastic/elastic-package/releases
2022/08/10 19:54:56 DEBUG running command: /home/mariorodriguez/bin/docker-compose version --short
2022/08/10 19:54:56 DEBUG Services to check: [elasticsearch elastic-agent fleet-server kibana package-registry]
2022/08/10 19:54:56 DEBUG running command: /home/mariorodriguez/bin/docker-compose -f /home/mariorodriguez/.elastic-package/profiles/default/stack/snapshot.yml -p elastic-package-stack ps -q elasticsearch elastic-agent fleet-server kibana package-registry
no such service: elastic-agent
2022/08/10 19:54:56 ERROR is Elastic stack created/running?
Error: failed getting stack status: running command failed: exit status 1
```

Reply:
Latest commit changes how status command checks the services. With the latest commit:
1. Run `docker-compose ps -q` to get all services running
2. Filter out the services that contains "is_ready" in their names
3. Shows the information for all the remaining services

This works for both docker-compose versions v1 and v2